lakefile.lean

```diff
@@ -68,6 +68,7 @@ lean_lib Counterexamples where
 lean_lib ImportGraph where
   roots := #[`ImportGraph]
 
+@[default_target]
```

Comment:
`lake exe graph` is failing to compile on master. I've fixed the problem in this PR, and adding `@[default_target]` here ensures that `lake build` will catch this in future.

Reply:
Do I understand correctly that making it a default target will make it be built (compiled) for everyone who types lake build in mathlib? If that is the case what do you think of adding `lake build graph` as a separate CI step instead of this?

Reply:
This is that same as for `checkYaml` and `runLinter`. I'll undo this for now.

Reply:
Yeah I'd definitely argue those shouldn't be built by a bare `lake build` either. But thats a topic for another discussion!